packages/config/config/devices/0x010f/fgd212.json

```diff
@@ -26,21 +26,55 @@
 		}
 	],
 	"firmwareVersion": {
-		"min": "3.5",
+		"min": "0.0",
 		"max": "255.255"
 	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Send BASIC SET command class frame S1",
+			"maxNodes": 10
+		},
+		"3": {
+			"label": "Sends MULTILEVEL SWITCH command class frame S1",
+			"maxNodes": 10
+		},
+		"4": {
+			"label": "Sends BASIC SET command class S2",
+			"maxNodes": 10
+		},
+		"5": {
+			"label": "Sends MULTILEVEL SWITCH command class frame S2",
+			"maxNodes": 10
+		}
+	},
 	"paramInformation": [
 		{
 			"#": "1",
-			"label": "Minimum Brightness Level",
-			"description": "For manual override, this is set during auto-calibration",
+			"label": "Minimum brightness level",
```

Comment:
I missed that because they only cleaned up one of the fw split files. My script built off them in order and these changes weren't in the 0.0 file. From now on, we should make sure people do the entire thing.